ssh.go

```diff
@@ -74,6 +75,13 @@ func (t *sshSmartSubtransport) Action(urlString string, action SmartServiceActio
 		return nil, err
 	}
 
+	// Escape \ and '.
+	uPath := strings.ReplaceAll(u.Path, `\`, "\\")
+	uPath = strings.ReplaceAll(uPath, "'", "\\'")
```

Comment:
```suggestion
	uPath := strings.ReplaceAll(u.Path, `\`, `\\`)
	uPath = strings.ReplaceAll(u.Path, `'`, `\'`)
```

maybe it's easier to only use backticks to make it easier to understand: currently the first line is a no-op, because ``` `\` == "\\"``` ! source: https://play.golang.org/p/gVcwOh-VM-f

Reply:
Thanks for providing that example. Very helpful.
Updated the commit.